app/models/user.rb

```diff
@@ -17,4 +17,7 @@ class User < ApplicationRecord
   end
 
   validates :birth, presence: true, format: { with: /\d/ }
+
+  has_many :items
+  has_many :orders
```

Comment:
```suggestion
  #has_many :orders
```
こちらも上記のようにコメントアウトしておきましょう。
購入機能実装の際に差分を出すためです。

Reply:
コメントアウトしました